Description: <add sunway support>
Author: Miao Changwei <miaochangwei@uniontech.com>

---
Last-Update: 2023-10-07

--- a/build-aux/config.guess
+++ b/build-aux/config.guess
@@ -1143,6 +1143,9 @@
     sparc:Linux:*:* | sparc64:Linux:*:*)
 	GUESS=$UNAME_MACHINE-unknown-linux-$LIBC
 	;;
+    sw_64:Linux:*:*)
+	GUESS=$UNAME_MACHINE-unknown-linux-$LIBC
+	;;
     tile*:Linux:*:*)
 	GUESS=$UNAME_MACHINE-unknown-linux-$LIBC
 	;;
--- a/build-aux/config.sub
+++ b/build-aux/config.sub
@@ -1266,6 +1266,7 @@
 			| sparc | sparc64 | sparc64b | sparc64v | sparc86x | sparclet \
 			| sparclite \
 			| sparcv8 | sparcv9 | sparcv9b | sparcv9v | sv1 | sx* \
+			| sw_64 \
 			| spu \
 			| tahoe \
 			| thumbv7* \
